sp/src/game/server/hl2/hl2_player.h

```diff
@@ -270,6 +270,7 @@ class CHL2_Player : public CBasePlayer
 
 #ifdef MAPBASE
 	virtual bool		CanAutoSwitchToNextBestWeapon( CBaseCombatWeapon *pWeapon );
+	int					m_nLastWaterLevel;
```

Comment:
Why is this placed here instead of with the other variables in `CHL2_Player`? Shouldn't it be saved as well?

Reply:
If a player saves their game and reloads it, wouldn't the outputs fire again because the game doesn't record `m_nLastWaterLevel`?

Reply:
I see what you mean now.
Yes. But _only_ if the save was done while being submerged. This appears to be an oversight of mine